ambari-server/src/main/java/org/apache/ambari/server/controller/AmbariServer.java

```diff
@@ -200,6 +200,7 @@
    */
   public static final EnumSet<DispatcherType> DISPATCHER_TYPES = EnumSet.of(DispatcherType.REQUEST);
   private static final int DEFAULT_ACCEPTORS_COUNT = 1;
+  private static final String[] DEPRECATED_SSL_PROTOCOLS = new String[] {"TLSv1"};
```

Comment:
Can this be made configurable in `ambari.properties` such that user can also define their exclude list on top of this deprecated list?

Reply:
The such deprecation list is already implemented with "security.server.disabled.protocols" property.